### PR TITLE
switch to use the kube proxy to open the dashboard, so we can support 1.16/1.17.

### PR DESCRIPTION
**Pull request by andyliuliming**
… 1.16/1.17.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### CODE REVIEW

File: src/aks-preview/azext_aks_preview/custom.py [hunk 1]
```diff
@@ -585,9 +585,9 @@ def aks_browse(cmd,     # pylint: disable=too-many-statements
         protocol = 'https'
     else:
         protocol = 'http'
-
-    proxy_url = '{0}://{1}:{2}/'.format(protocol, listen_address, listen_port)
-
+    
+    proxy_url = 'http://{1}:{2}/'.format(protocol, listen_address, listen_port)
```
Comment (bingosummer) [inline src/aks-preview/azext_aks_preview/custom.py]: => proxy_url = 'http://{0}:{1}/'.format(listen_address, listen_port)

File: src/aks-preview/azext_aks_preview/custom.py [hunk 2]
```diff
@@ -586,8 +586,9 @@ def aks_browse(cmd,     # pylint: disable=too-many-statements
     else:
         protocol = 'http'
 
-    proxy_url = '{0}://{1}:{2}/'.format(protocol, listen_address, listen_port)
-
+    proxy_url = 'http://{1}:{2}/'.format(listen_address, listen_port)
```
Comment (zqingqing1) [inline src/aks-preview/azext_aks_preview/custom.py]: http://{0}:{1}/

Comment (zqingqing1) [inline src/aks-preview/azext_aks_preview/custom.py]: http://{0}:{1}

**Comment by qwordy** [comment]
@zqingqing1 Could you review again? Thanks

**Comment by andyliuliming** [comment]
@qwordy could you please help merge this?

**Comment by qwordy** [comment]
> 
> 
> @qwordy could you please help merge this?

sure

**Comment by andyliuliming** [comment]
@qwordy ping :)

**Comment by yungezz** [comment]
hi @andyliuliming the PR need to merge from master. I will merge it after the branch updating.

**Comment by qwordy** [comment]
> 
> 
> @qwordy ping :)

When I try to merge this PR, it says it's out of date. So I merge from master branch and trigger CI. It takes some time. Then I forget this PR. Sorry about that.

**Pull request merged**